packages/salesforcedx-vscode-apex/package.nls.json

```diff
@@ -6,6 +6,7 @@
   "refresh_test_title": "Refresh Tests",
   "run_single_test_title": "Run Single Test",
   "show_error_title": "Display Error",
+  "show_definition_title": "Go to Definition",
```

Comment:
"show_definition_title" -> "go_to_definition_title"?

Reply:
2. Right, you'll only ever see "Go to Definition" or "Display Error"
3. Yeah I should be consistent, I'll update that.